packages/expo-ads-admob/src/AdMobInterstitial.ts

```diff
@@ -38,11 +39,10 @@ export default {
     await AdMobNativeModule.setAdUnitID(id);
   },
   async setTestDeviceID(id: string): Promise<void> {
-    if (!AdMobNativeModule.setTestDeviceID) {
-      throw new UnavailabilityError(moduleName, 'setTestDeviceID');
-    }
-
-    await AdMobNativeModule.setTestDeviceID(id);
+    console.warn(
+      'AdMobInterstitial.setTestDeviceID is deprecated. Test device IDs are now set globally. Use AdMob.setTestDeviceID instead.'
+    );
+    setTestDeviceID(id);
```

Comment:
```suggestion
    await setTestDeviceID(id);
```
Suggestion valid only if we decide not to upgrade Android in the same PR